gestion_base_adresse/processing/structure/upgrade_database_structure.py

```diff
@@ -213,29 +209,40 @@ def getKey(item):
                 # Add SQL database version in adresse.metadata
                 new_db_version = sf.replace('upgrade_to_', '').replace('.sql', '').strip()
                 feedback.pushInfo('* NOUVELLE VERSION BDD ' + new_db_version )
-                sql+= '''
+                sql += '''
                     UPDATE adresse.metadata
                     SET (me_version, me_version_date)
                     = ( '%s', now()::timestamp(0) );
                 ''' % new_db_version
 
-                [header, data, rowCount, ok, error_message] = fetchDataFromSqlQuery(
+                _, _, _, ok, error_message = fetchDataFromSqlQuery(
                     connection_name,
                     sql
                 )
-                if ok:
-                    feedback.pushInfo('* ' + sf + ' -- OK !')
-                else:
+                if not ok:
                     feedback.reportError(error_message)
-                    status = 0
-                    raise Exception(error_message)
-                    # return {
-                        # self.OUTPUT_STATUS: status,
-                        # self.OUTPUT_STRING: error_message
-                    # }
+                    raise QgsProcessingException(error_message)
+
+                feedback.pushInfo('* ' + sf + ' -- OK !')
+
+        # Everything is fine, we now update to the plugin version
+        sql += '''
```

Comment:
Juste use `sql = ` instead of `sql += ` I think, since we do not want to replay previous SQL

Reply:
Oh yes !